__tests__/component-options.test.js

```diff
@@ -62,24 +51,24 @@ describe('Component page', () => {
   })
 
   it('macro options should be opened and in view when linked to', async () => {
-    await page.goto(baseUrl + '/components/back-link/#options-back-link-example', { waitUntil: 'load' })
+    await goTo(page, '/components/back-link/#options-back-link-example')
 
     // Check if example's macro options details element is open
     await page.waitForSelector('#options-back-link-example-details[open=open]')
 
     // Check if the example has been scrolled into the viewport
-    const $example = await page.$('#options-back-link-example')
-    expect(await $example.isIntersectingViewport()).toBe(true)
+    const $example = await page.$('#options-back-link-example-details')
```

Comment:
Had to swap this to the details element since `#options-back-link-example` is an empty `<span>`

```html
<span id="options-back-link-example"></span>
```

Perhaps due to its zero size it no longer passes `.isIntersectingViewport()` in Puppeteer v18